stories/SearchSelectField/SearchSelectField.stories.js

```diff
@@ -130,7 +130,8 @@ storiesOf('search select field', module)
             'kitten',
             'puppy',
             'duck',
-            'capybara'
+            'capybara',
+            'Chompsky the only Brussels Griffon Chompsky the only Brussels Griffon'
```

Comment:
It will never be wrong to include Chompsky in test text :-D

Reply:
Agreed.  :)